CHANGELOG.md

```diff
@@ -4,8 +4,11 @@
 
   **FIXED**
   - Mastering workflow example was missing the generate clusters step, which has been rectified using proper endpoint
+  - `data` field renamed to `_data` (private). ([details](https://github.com/Datatamer/unify-client-python/issues/61))
```

Comment:
The removal of `.data` should be noted under a `**REMOVED**` header as well, since it will be a breaking change for those are currently using it.

nit: could you also rename your link from `details` to `PR`? We originally went with `details` in past changelog entries but that was too vague. sometimes we want to include the issue #, sometimes the PR #, and sometimes both.